x-pack/plugins/security_solution/public/common/components/wrapper_page/index.tsx

```diff
@@ -8,13 +8,14 @@ import classNames from 'classnames';
 import React, { useEffect } from 'react';
 import styled from 'styled-components';
 
+import { CommonProps } from '@elastic/eui/src/components/common';
 import { useFullScreen } from '../../containers/use_full_screen';
 import { gutterTimeline } from '../../lib/helpers';
 import { AppGlobalStyle } from '../page/index';
 
-const Wrapper = styled.div`
-  padding: ${({ theme }) =>
-    `${theme.eui.paddingSizes.l} ${gutterTimeline} ${theme.eui.paddingSizes.l} ${theme.eui.paddingSizes.l}`};
+const Wrapper = styled.div<{ noTimeline?: boolean }>`
```

Comment:
Make sure you get a reviewer from SIEM to 👍 this 😬

Reply:
FYI @XavierM or @spong thoughts on this?

Reply:
Cleaned up the code a bit there, but yes would be great to get review from authors.

Reply:
that looks nice!